wordpress/wp-content/themes/_bbr/page-books.php

```diff
@@ -1,17 +1,50 @@
 <?php
 /*
-Template Name: Books list
+Template Name: Book List
 */
 
-get_header();
-?>
+get_header(); ?>
 
-<div id="books" class="content full"></div>
-
-<script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/3.8.0/lodash.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-<script src="<?= get_template_directory_uri() ?>/js/react-0.13.2.min.js"></script>
-<script src="<?= get_template_directory_uri() ?>/js/ReactRouter.min.js"></script>
-<script src="<?= get_template_directory_uri() ?>/js/books.js"></script>
+<div class="container">
+  <div class="row">
+    <div class="col-md-3 sidebar right-sidebar">
+      <div class="widget sidebar-widget box-style1">
+        <!-- TODO: figure out what is supposed to go over here, maybe nothing? -->
+        <h3 class="widget-title">Top categories</h3>
+        <ul class="top-categories-list">
+        </ul>
+      </div>
+      <div class="widget sidebar-widget box-style1">
+        <h3 class="widget-title">Categories</h3>
+        <ul class="categories-list">
+          <!-- TODO: not sure what this is for either -->
```

Comment:
This is where the keywords (book tags, after they're ported, I believe) will go.